qdrant_client/local/sparse.py

```diff
@@ -24,18 +28,20 @@ def is_sorted(vector: SparseVector) -> bool:
     return True
 
 
-def sort(vector: SparseVector) -> SparseVector:
+def sort_sparse_vector(vector: SparseVector) -> SparseVector:
     if is_sorted(vector):
         return vector
 
     sorted_indices = np.argsort(vector.indices)
     return SparseVector(
-        indices=vector.indices[sorted_indices],
-        values=vector.values[sorted_indices],
+        indices=[vector.indices[i] for i in sorted_indices],
```

Comment:
Why is this change necessary?

Reply:
`vector.indices` and `vector.values` are built-in python lists
This kind of lists does not support indexing by array of indices

To take elements from an array by several indices, this array should support such an operation (e.g. np.array)